pages/documentation/documentation.html

```diff
@@ -8,15 +8,15 @@ <h2>OData Version 4.01</h2>
-<li><a href="http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html">Part 2: URL Conventions</a> | <a href="http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.pdf">PDF</a></li>
-<li><a href="http://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.html">OData JSON Format</a> | <a href="http://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.pdf">PDF</a></li>
-<li><a href="http://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html">OData Common Schema Language (CSDL) XML Representation</a> | <a href="http://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.pdf">PDF</a></li>
-<li><a href="http://docs.oasis-open.org/odata/odata-csdl-json/v4.01/odata-csdl-json-v4.01.html">OData Common Schema Language (CSDL) JSON Representation</a> | <a href="http://docs.oasis-open.org/odata/odata-csdl-json/v4.01/odata-csdl-json-v4.01.pdf">PDF</a></li>
+<li><a href="http://docs.oasis-open.org/odata/new-in-odata/v4.01/new-in-odata-v4.01.html">What’s New in OData Version 4.01</a> | <a href="http://docs.oasis-open.org/odata/new-in-odata/v4.01/new-in-odata-v4.01.pdf" aria-label="'What's new in ODATA version 4.01 PDF'">PDF</a></li>
+<li><a href="http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html">Part 1: Protocol</a> | <a href="http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.pdf" aria-label="'Part 1: Protocol' PDF">PDF</a></li>
+<li><a href="http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html">Part 2: URL Conventions</a> | <a href="http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.pdf" aria-label="'Part 2: URL Conventions' PDF">PDF</a></li>
+<li><a href="http://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.html">OData JSON Format</a> | <a href="http://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.pdf" aria-label="'OData JSON Format' PDF">PDF</a></li>
+<li><a href="http://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html">OData Common Schema Language (CSDL) XML Representation</a> | <a href="http://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.pdf" aria-label="'OData Common Schema Language (CSDL) XML Representation' PDF">PDF</a></li>
+<li><a href="http://docs.oasis-open.org/odata/odata-csdl-json/v4.01/odata-csdl-json-v4.01.html">OData Common Schema Language (CSDL) JSON Representation</a> | <a href="http://docs.oasis-open.org/odata/odata-csdl-json/v4.01/odata-csdl-json-v4.01.pdf" aria-label="'OData Common Schema Language (CSDL) JSON Representation' PDF">PDF</a></li>
 <li><a href="http://docs.oasis-open.org/odata/odata/v4.01/cs01/abnf/odata-abnf-construction-rules.txt">OData ABNF</a></li>
```

Comment:
Note we're still a bit inconsistent -- with the first labels, the closing single quote is before the PDF. In the later labels, the PDF is outside of the single quotes.